roofit/roofitcore/inc/RooAdaptiveIntegratorND.h

```diff
@@ -52,8 +52,8 @@ class RooAdaptiveIntegratorND : public RooAbsIntegrator {
 
   bool _useIntegrandLimits;  ///< If true limits of function binding are used
 
```

Comment:
same comment as above, although maybe this class is not really stored in a file

Reply:
This class has version number zero `ClassDefOverride(RooAdaptiveIntegratorND,0)`, so we don't need to consider IO here.